### ISSUE TITLE
Impact of the pytorch version on the performance

### ISSUE THREAD

**Issue by Shengqiang-Li** [espnet/espnet#3242]
I have achieved the training and decoding on AISHELL-1 dataset using torch1.8+cu111 and python 3.8, and got a cer of 4.7%/5.1% on dev/test set, which is significantly worse than the reported results.  Is that caused by the version of pytorch and python?

**Comment by sw005320** [comment]
Maybe yes, but it also depends on the GPU type, cudnn versions, etc, and it is very difficult for us to mention.
A model in zenodo https://zenodo.org/record/4105763#.YK6xfpNKhH1 also includes the learning curve and so on, and you can compare more detailed learning behaviors with yours.

**Comment by Shengqiang-Li** [comment]
Thank you very much. I have another question, the recipe of AISHELL-1 and LibriSpeech have a difference between their environment. Now I want to conduct experiments on both AISHELL-1 and LibriSpeech, can I unify the pytorch version and python version for AISHELL-1 and LibriSpeech?

**Comment by sw005320** [comment]
I recommend you unify the PyTorch version to make your experiments consistent.
